Presentation.Web/Controllers/External/V2/ItInterfaceV2Controller.cs

```diff
@@ -32,18 +47,33 @@ public IHttpActionResult PostItInterface([FromBody] ItInterfaceRequestDTO itInte
         /// <summary>
         /// Returns active IT-Interfaces
         /// </summary>
+        /// <param name="organizationUuid">Uuid of the organization you want interfaces from</param>
         /// <param name="page">Page index to be returned (zero based)</param>
         /// <param name="pageSize">Page size</param>
         /// <returns></returns>
         [HttpGet]
-        [Route("it-interfaces")]
+        [Route("rightsholder/it-interfaces")]
         [SwaggerResponse(HttpStatusCode.OK, Type = typeof(IEnumerable<ItInterfaceResponseDTO>))]
         [SwaggerResponse(HttpStatusCode.BadRequest)]
         [SwaggerResponse(HttpStatusCode.Unauthorized)]
         [SwaggerResponse(HttpStatusCode.Forbidden)]
-        public IHttpActionResult GetItInterface(int? page = 0, int? pageSize = 100)
+        public IHttpActionResult GetItInterface(Guid? rightsHolderUuid = null, [FromUri] StandardPaginationQuery pagination = null)
         {
-            return Ok(new List<ItInterfaceResponseDTO>());
+            if (!ModelState.IsValid)
+                return BadRequest(ModelState);
+
+            var refinements = new List<IDomainQuery<ItInterface>>();
+
+            if (rightsHolderUuid.HasValue)
+                refinements.Add(new QueryByRightsHolder(rightsHolderUuid.Value));
+
+            return _itInterfaceService
+                .GetAvailableInterfaces(refinements.ToArray())
```

Comment:
Lav en metode i IRightsHoldersService som tjekker at du har lov (er rettighedshaver) og den kalder så videre ned i interface servicen.

I den kan du så tjekke på "Er du rettighedshaver i queryparamorg? -> not -> forbidden. Any org -> not? forbidden , else -> kald interface service som den er i dag.